flexget/plugins/urlrewrite_divxatope.py

```diff
@@ -8,24 +8,59 @@
 from flexget.utils import requests
 from flexget.utils.soup import get_soup
 
+from flexget.entry import Entry
+from flexget.utils.search import torrent_availability, normalize_unicode
+
 log = logging.getLogger('divxatope')
 
+CATEGORIES = {
```

Comment:
So you want to make it 

```
divxatope: yes/no
```

If so then something like this might work (untested)

```
schema = {
        'type': 'boolean',
        'default': False
    }
```

You will also need to check it its disabled during search by checking

```
if config:
   # do stuff
```